packages/kbn-utils/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/utils
+
+Shared server-side utilities shared across packages and plugins.
```

Comment:
PR title states that PR closes https://github.com/elastic/kibana/issues/76925
However, `src/core/utils` contains methods extending js standard library and they aren't nodejs specific. Are we going to move `src/core/utils` under a different package, `@kbn/std` as proposed in https://github.com/elastic/kibana/issues/76925#issuecomment-689096491? Then we should keep the parent issue open.